app/src/main/java/com/github/se/eduverse/ui/profile/ProfileScreen.kt

```diff
@@ -190,12 +195,110 @@ private fun StatItem(label: String, count: Int, modifier: Modifier = Modifier) {
   }
 }
 
+@Composable
+private fun PublicationItem(
+    publication: Publication,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+  Card(
+      modifier = modifier.aspectRatio(1f).clickable(onClick = onClick),
+      shape = RoundedCornerShape(8.dp)) {
+        Box(modifier = Modifier.testTag("publication_content_${publication.id}").fillMaxSize()) {
+          AsyncImage(
+              model =
+                  ImageRequest.Builder(LocalContext.current)
+                      .data(publication.thumbnailUrl)
+                      .crossfade(true)
+                      .listener(
+                          onError = { _, _ ->
+                            println("Failed to load thumbnail for ${publication.id}")
+                          },
+                          onSuccess = { _, _ ->
+                            println("Successfully loaded thumbnail for ${publication.id}")
+                          })
+                      .build(),
+              contentDescription = "Publication thumbnail",
+              modifier = Modifier.testTag("publication_thumbnail_${publication.id}").fillMaxSize(),
+              contentScale = ContentScale.Crop,
+              error = painterResource(R.drawable.eduverse_logo_alone),
+              fallback = painterResource(R.drawable.eduverse_logo_alone),
+          )
+
```

Comment:
Using painterResource(R.drawable.eduverse_logo_alone) as fallback and error image is correct, but it might be more relevant to use specific resources to distinguish these two cases (e.g. a different error icon).